.github/workflows/ci.yml

```
@@ -2,42 +2,49 @@ name: CI
-    - name: Type checking
-      run: pnpm run build
+      - name: Linting
+        run: pnpm run lint
 
-    - name: Linting
-      run: pnpm run lint
+      - name: Run tests
+        run: pnpm test
+
+      - name: Upload coverage reports
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)